src/components/TodoError/TodoError.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useContext } from 'react';
+import cn from 'classnames';
+
+import { TodosContext } from '../TodosContext';
+
+export const TodoError: React.FC = () => {
+  const { todoError, setTodoError } = useContext(TodosContext);
```

Comment:
You can create a custom hook for your context and avoid passing every time the same context into `useContext`
```javascript
const useTodos = () => useContext(TodosContext);
```